### PR TITLE
Added new user profile json file

### PR DESCRIPTION
**Pull request by Shukla007**

### CODE REVIEW

File: public/data/shukla007.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "http://twitter.com/avi0_07",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhishek-shukla-682908211/",
+        "icon": "linkedin"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/shukla007.json:22]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/shukla007.json (hunk 1 above)
Comment (Shukla007) [reply public/data/shukla007.json:22]: It looks fine to me can explain it what do mean by missing an endline

File: public/data/shukla007.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "url": "http://twitter.com/avi0_07",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhishek-shukla-682908211/",
+        "icon": "linkedin"
+      }
+    ]
+
```
Comment (schmelto) [inline public/data/shukla007.json:22]:
```suggestion
```

Comment (Shukla007) [reply public/data/shukla007.json:22]: what does profile addition mean and what changes are required.

Comment (mohan-murali) [reply public/data/shukla007.json:22]: LinkFree has profiles of lots of user. Profile addition tag allows the maintainers to know that this is a PR for addition of profile. The changes requested here are to remove the empty line (line no 22)

Comment (Shukla007) [reply public/data/shukla007.json:22]: okk Thanks a lot

**Review by krishguptadev — APPROVED** [review]

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Do what @schmelto suggested 
Rest looks fine to me

**Comment by Shukla007** [comment]
I have already changed what @schmelto suggested

**Review by Pradumnasaraf — APPROVED** [review]

File: public/data/shukla007.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhishek-shukla-682908211/",
+        "icon": "linkedin"
+      }
+    ]
+}
+
```
Comment (Panquesito7) [inline public/data/shukla007.json:23]: Now there are two endlines, as the Git diff doesn't show the last one. 😅
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM! 🚀

**Review by mohan-murali — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]

**Pull request merged**